### DISCUSSION TITLE
Manual installation example

### DISCUSSION THREAD

**Discussion by RuudKapteijn** [dylandoamaral/uptime-card#151] [Q&A] [answered]
This is a beautiful card. I would like to use it.

I am struggeling with the configuration. Would it be possible to provide more details on the manual installation and configuration of the card? A code example of this instruction would be helpful: "Add the resource reference inside your configuration.yaml with URL /local/uptime-card.js and type module".

Kind regards, Ruud.

**Comment by dylandoamaral** [comment, marked as answer]
HI Ruud, first of all, did you consider using [HACS](https://hacs.xyz/) ? It will make the process much more simple. I honestly installed it by myself just once before using HACS. 

Nevertheless if you still want to register the card by yourself, you have to add the resource reference:
- Inside the UI if you don't use a YAML file following this documentation: https://developers.home-assistant.io/docs/frontend/custom-ui/registering-resources/
- Inside the configuration.yaml if you do use a YAML file writing the following piece of code:
```
resources:
  - type: module
    url: /local/uptime-card.js
```

If you have any trouble don't hesitate. Also don't hesitate to make a pull request if you think the README file should be more precise.

**Comment by RuudKapteijn** [comment]
Hi Dylan, thank you for your help. I am a newbie with respect to custom cards and Lovelace dashboards.

Following the instructions you pointed out, I have registered the resource (using HACS). That worked just fine. I also configured the card:

Kind regards, Ruud.

![image](https://user-images.githubusercontent.com/44146047/220209671-ba8a43f4-c1e1-45a0-a65e-2e94cbdd2dc1.png)
